src/utils/link-types.js

```diff
@@ -10,3 +10,5 @@ const linkTypes = [LINK, AT_LINK, LOCAL_LINK, EMAIL, HASHTAG, ARROW];
 export function isLink({type}) {
   return linkTypes.indexOf(type) !== -1;
 }
+
+export const FRIENDFEED_POST = new RegExp(`^http://friendfeed.com/[^/]+/([0-9a-f]{8})(?:/|$)`);
```

Comment:
There is no https:// or schema-less Friendfeed URLs in archive posts. These URLs was processed and normalized by Friendfeed itself and these are all in form http://friendfeed.com/…